src/addon.cpp

```diff
@@ -66,35 +66,39 @@ class SHA3Hash: public ObjectWrap {
 
 	static
 	NAN_MODULE_INIT(Init) {
-		Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
-		t->SetClassName(Nan::New<String>("SHA3Hash").ToLocalChecked());
-		t->InstanceTemplate()->SetInternalFieldCount(1);
+		Local<Context> context = target->CreationContext();
```

Comment:
:memo: **Implementation Note:** This `context` is now required for `GetFunction()` down below.